assemblyline/common/classification.py

```diff
@@ -123,7 +123,12 @@ def __init__(self, classification_definition: Dict):
                 self.groups_map_lts[name] = short_name
                 self.groups_map_stl[short_name] = name
                 for a in x.get('aliases', []):
-                    self.groups_aliases[a.upper()] = self.groups_aliases.get(a.upper(), []) + [short_name]
+                    self.groups_aliases[a.upper()] = \
+                        list(set(self.groups_aliases.get(a.upper(), []) + [short_name]))
+                solitary_display_name = x.get('solitary_display_name', None)
```

Comment:
Entirely an aesthetic thought that I don't feel strongly about but: this line could be 
`solitary_display_name = x.get('solitary_display_name', '').upper()`
And the double use of `solitary_display_name.upper()` wouldn't need the method call there